lsp-dls.el

```diff
@@ -0,0 +1,56 @@
+;;; lsp-dls.el --- dls configuration               -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 lsp-mode developers
+
+;; Author: Hiroki Noda <kubo39@gmail.com>
+;; Keywords:
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; dls client configuration
+
+;;; Code:
+
+(require 'lsp-mode)
+
+(defgroup lsp-dls nil
+  "LSP support for D, using https://github.com/d-language-server/dls."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/d-language-server/dls"))
+
+(defcustom lsp-dls-path "~/.dub/packages/.bin/dls-latest/dls"
```

Comment:
I think it should be just `dls` here, since even in other language servers, it would just expose the server binary to `$PATH`.
That will help with user who install server into custom path and then expose the server to global PATH